### PR TITLE
remove work with FileService

### PR DESCRIPTION
**Pull request by VLiamin**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/901

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -56,6 +56,7 @@ dotnet_naming_rule.constant_fields_should_be_pascal_case.style    = pascal_case_
 dotnet_naming_symbols.constant_fields.applicable_kinds            = field
 dotnet_naming_symbols.constant_fields.applicable_accessibilities  = *
 dotnet_naming_symbols.constant_fields.required_modifiers          = const
+tab_width=2
```
Comment (BelehovEgor) [inline .editorconfig:59]: we don't use tab size, we use indentation size

File: src/CompanyService.Mappers/Models/CompanyInfoMapper.cs [hunk 2]
```diff
@@ -1,58 +1,59 @@
...
-            IDepartmentInfoMapper departmentMapper,
-            IPositionInfoMapper positionMapper,
-            IOfficeInfoMapper officeMapper)
-        {
-            _departmentMapper = departmentMapper;
-            _positionMapper = positionMapper;
-            _officeMapper = officeMapper;
-        }
+    public CompanyInfoMapper(
+        IDepartmentInfoMapper departmentMapper,
```
Comment (BelehovEgor) [inline src/CompanyService.Mappers/Models/CompanyInfoMapper.cs:16]: fix

**Review by BelehovEgor — APPROVED** [review]

File: .editorconfig (hunk 1 above)
Comment (VLiamin) [reply .editorconfig]: done

File: src/CompanyService.Business/Commands/Company/CreateCompanyCommand.cs [hunk 3]
```diff
@@ -32,26 +33,26 @@ public class CreateCompanyCommand : ICreateCompanyCommand
...
             port: smtpInfo.Port,
             enableSsl: smtpInfo.EnableSsl,
             email: smtpInfo.Email,
-            password: smtpInfo.Password)).Result.Message;
+            password: smtpInfo.Password));
 
-        if (response.IsSuccess && response.Body)
+        if (response.Message.IsSuccess && response.Message.Body)
         {
           return true;
```
Comment (MariyaSin) [inline src/CompanyService.Business/Commands/Company/CreateCompanyCommand.cs:52]: you can remove response.Message.Body from 50 string and return it (return response.Message.Body)

File: src/CompanyService.Business/Commands/Company/CreateCompanyCommand.cs [hunk 4]
```diff
@@ -148,8 +149,6 @@ public OperationResultResponse<Guid> Execute(CreateCompanyRequest request)
 
       _repository.Add(company);
 
-      //TODO async
-      //Task.Run(() =>
```
[the remark this answers is not in the document]
Comment (VLiamin) [reply src/CompanyService.Business/Commands/Company/CreateCompanyCommand.cs:152]: done

File: src/CompanyService.Validation/Company/EditCompanyRequestValidator.cs [hunk 5]
```diff
@@ -1,121 +1,135 @@
...
+      AddСorrectOperations(nameof(EditCompanyRequest.CompanyName), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditCompanyRequest.Description), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditCompanyRequest.SiteUrl), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditCompanyRequest.Host), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditCompanyRequest.Port), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditCompanyRequest.Email), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditCompanyRequest.Password), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditCompanyRequest.EnableSsl), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditCompanyRequest.Logo), new List<OperationType> { OperationType.Replace, OperationType.Add, OperationType.Remove });
```
Comment (MariyaSin) [inline src/CompanyService.Validation/Company/EditCompanyRequestValidator.cs:53]: only replace

**Review by MariyaSin — APPROVED** [review]

File: src/CompanyService.Validation/Company/EditCompanyRequestValidator.cs (hunk 5 above)
Comment (VLiamin) [reply src/CompanyService.Validation/Company/EditCompanyRequestValidator.cs]: done

**Pull request merged**